Frameworks/UIKit/UIApplication.mm

```diff
@@ -423,234 +332,131 @@ + (UIApplication*)sharedApplication {
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
 */
 - (void)scheduleLocalNotification:(UILocalNotification*)n {
     UNIMPLEMENTED();
-    [n _setReceiver:self];
-    int idx = [_curNotifications indexOfObject:n];
-    if (idx == NSNotFound) {
-        [_curNotifications addObject:n];
-    }
 }
 
-- (void)_receiveAlarm:(id)localNotification {
-    int idx = [_curNotifications indexOfObject:localNotification];
-    if (idx == NSNotFound) {
-        return; // probably deleted.
-    }
-
-    UIAlertView* alert = [[UIAlertView alloc] initWithTitle:@"Alert"
-                                                    message:[localNotification alertBody]
-                                                   delegate:nil
-                                          cancelButtonTitle:[localNotification alertAction]
-                                          otherButtonTitles:nil];
+/**
+ @Status NotInPlan
+ @Notes UWP status bar can be directly accessed via WinRT APIs, and will eventually be supported via UIViewController APIs.
```

Comment:
No, these APIs are deprecated.  We'll eventually implement them on UIViewController (if anywhere).

---
In reply to: [110294530](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r110294530) [](ancestors = 110294530)